story.py

```diff
@@ -1,13 +1,26 @@
 import csv
 
 def read_tsv():
-	# read tsv and print rows
-
+	prev_choices_dict = {}
+	next_choices_dict = {}
+	stories = {}
```

Comment:
You want to be able to access these from your `user_input()` function to code the story flow, so they need to be global variables.